configure.ac

```diff
@@ -25,7 +25,7 @@ AC_PREREQ([2.69])
 
 # Initialize with name, version, and support email address.
 AC_INIT([GFDL FMS Library],
-  [2021.02.0],
+  [2021.02.0-dev],
```

Comment:
With 2021.02 minted, should this be 2021.03.0-dev?

Reply:
In the past it's just been appended to the previous version number like this, but the other way definitely makes more sense as it should be the development version for the next release. Should we start naming it this way moving forward?